source/security.rst

```diff
@@ -66,13 +77,17 @@ Create a new key::
     SHA256: [...] gert@hostname
     The key's randomart image is: [...]
 
-Using the same passphrase as your existing key(s) lets you add them all with a single use of `ssh-add`::
+Using the same passphrase as your existing key(s) lets you add them all with a single use of `ssh-add`.
+If all your keys are named
```

Comment:
I think these sentences should be combined, since "if all your keys..." qualifies the original and then you can drop "you can just use ..."